DemoBoolean.java

```diff
@@ -0,0 +1,11 @@
+package com.edocti.jintro.lab01;
```

Comment:
This is the same class as you created in another branch. You can delete it from this branch. It will appear in master anyway.